website_sale_secondary_unit/static/src/js/website_sale_secondary_unit.js

```diff
@@ -0,0 +1,100 @@
+odoo.define("website_sale_secondary_unit.animation", function(require) {
+    "use strict";
+
+    const sAnimation = require("website.content.snippets.animation");
+
+    sAnimation.registry.sale_secondary_unit = sAnimation.Class.extend({
+        selector: ".secondary-unit",
+        // eslint-disable-next-line no-unused-vars
+        init: function(parent, editableMode) {
+            this._super.apply(this, arguments);
+            this.$secondary_uom = null;
+            this.$secondary_uom_qty = null;
+            this.$product_qty = null;
+            this.secondary_uom_qty = null;
+            this.secondary_uom_factor = null;
+            this.product_uom_factor = null;
+            this.product_qty = null;
+        },
+        start: function() {
+            this.$secondary_uom = $("#secondary_uom");
+            this.$secondary_uom_qty = $(".secondary-quantity");
+            this.$product_qty = $(".quantity");
+            this._setValues();
+            this.$target.on("change", ".secondary-quantity", () => {
+                this._onChangeSecondaryUom();
+            });
+            this.$target.on("change", "#secondary_uom", () => {
+                this._onChangeSecondaryUom();
+            });
+            this.$product_qty.on("change", null, () => {
+                this._onChangeProductQty();
+            });
```

Comment:
You can call directly binding the context:
```suggestion
            this.$target.on("change", ".secondary-quantity", this._onChangeSecondaryUom.bind(this));
            this.$target.on("change", "#secondary_uom", this._onChangeSecondaryUom.bind(this));
            this.$product_qty.on("change", null, this._onChangeProductQty.bind(this));
```